alicloud/resource_alicloud_kvstore_account.go

```diff
@@ -0,0 +1,247 @@
+			if IsExceptedErrors(err, OperationDeniedDBStatus) {
+				return resource.RetryableError(err)
+			}
+			return resource.NonRetryableError(err)
+		}
+		addDebug(request.GetActionName(), raw, request.RpcRequest, request)
+		return nil
+	})
+
+	if err != nil {
+		return WrapErrorf(err, DefaultErrorMsg, "alicloud_kvstore_account", request.GetActionName(), AlibabaCloudSdkGoERROR)
+	}
+
+	d.SetId(fmt.Sprintf("%s%s%s", request.InstanceId, COLON_SEPARATED, request.AccountName))
+
+	if err := kvstoreService.WaitForKVstoreAccount(d.Id(), Available, DefaultTimeoutMedium); err != nil {
+		return WrapError(err)
+	}
+
+	return resourceAlicloudKVStoreAccountRead(d, meta)
+}
+
+func resourceAlicloudKVStoreAccountRead(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*connectivity.AliyunClient)
+	kvstoreService := KvstoreService{client}
+	object, err := kvstoreService.DescribeKVstoreAccount(d.Id())
+	if err != nil {
+		if NotFoundError(err) {
+			d.SetId("")
+			return nil
+		}
+		return WrapError(err)
+	}
+
+	d.Set("instance_id", object.InstanceId)
+	d.Set("name", object.AccountName)
+	d.Set("type", object.AccountType)
+	d.Set("description", object.AccountDescription)
+
+	return nil
+}
+
+func resourceAlicloudKVStoreAccountUpdate(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*connectivity.AliyunClient)
+	kvstoreService := KvstoreService{client}
+	d.Partial(true)
+	parts := strings.Split(d.Id(), COLON_SEPARATED)
+	instanceId := parts[0]
+	accountName := parts[1]
+
+	if d.HasChange("description") {
+		if err := kvstoreService.WaitForKVstoreAccount(d.Id(), Available, DefaultTimeoutMedium); err != nil {
+			return WrapError(err)
+		}
+		request := r_kvstore.CreateModifyAccountDescriptionRequest()
+		request.RegionId = client.RegionId
+		request.InstanceId = instanceId
+		request.AccountName = accountName
+		request.AccountDescription = d.Get("description").(string)
+
+		raw, err := client.WithRkvClient(func(rkvClient *r_kvstore.Client) (interface{}, error) {
+			return rkvClient.ModifyAccountDescription(request)
+		})
+		if err != nil {
+			return WrapErrorf(err, DefaultErrorMsg, d.Id(), request.GetActionName(), AlibabaCloudSdkGoERROR)
+		}
+		addDebug(request.GetActionName(), raw, request.RpcRequest, request)
+		d.SetPartial("description")
+	}
+
+	if d.HasChange("password") || d.HasChange("kms_encrypted_password") {
+		if err := kvstoreService.WaitForKVstoreAccount(d.Id(), Available, DefaultTimeoutMedium); err != nil {
+			return WrapError(err)
+		}
+		request := r_kvstore.CreateResetAccountPasswordRequest()
+		request.RegionId = client.RegionId
+		request.InstanceId = instanceId
+		request.AccountName = accountName
+
+		password := d.Get("password").(string)
+		kmsPassword := d.Get("kms_encrypted_password").(string)
+
+		if password == "" && kmsPassword == "" {
+			return WrapError(Error("One of the 'password' and 'kms_encrypted_password' should be set."))
+		}
+
+		if password != "" {
+			d.SetPartial("password")
```

Comment:
d.SetPartial("password") move to line 218

Reply:
done